java/src/main/java/lib/Tracing.java

```diff
@@ -3,29 +3,37 @@
 import com.uber.jaeger.Configuration;
 import com.uber.jaeger.Configuration.ReporterConfiguration;
 import com.uber.jaeger.Configuration.SamplerConfiguration;
+import com.uber.jaeger.samplers.ConstSampler;
 import io.opentracing.Scope;
 import io.opentracing.SpanContext;
 import io.opentracing.Tracer;
 import io.opentracing.propagation.Format;
 import io.opentracing.propagation.TextMapExtractAdapter;
 import io.opentracing.tag.Tags;
 
-import java.util.HashMap;
 import javax.ws.rs.core.MultivaluedMap;
+import java.util.HashMap;
 
 public final class Tracing {
     private Tracing() {
     }
 
     public static com.uber.jaeger.Tracer init(String service) {
-        SamplerConfiguration samplerConfig = new SamplerConfiguration("const", 1);
-        ReporterConfiguration reporterConfig = new ReporterConfiguration(true, null, null, null, null);
-        Configuration config = new Configuration(service, samplerConfig, reporterConfig);
+        SamplerConfiguration samplerConfig = SamplerConfiguration.fromEnv()
+                .withType(ConstSampler.TYPE)
+                .withParam(1);
+
+        ReporterConfiguration reporterConfig = ReporterConfiguration.fromEnv()
```

Comment:
This is the only really required change to get it working on Katacoda. This is because I couldn't get the UDP sender to work in that environment and need to use the HttpSender, by overriding the endpoint env var.